src/components/reddit/RedditReturn.jsx

```diff
@@ -1,17 +1,26 @@
-import { useSelector } from "react-redux";
-import { selectRedditReturn } from "./RedditSlice";
 import Reddit from "./Reddit";
+import ListItemComponent from "../listItemComponent/ListItemComponent";
+
 
 const RedditReturn = () => {
-    const redditReturn = useSelector(selectRedditReturn);
+    const propsPassedIn = {
```

Comment:
Since you don't need to pass down props with redux, you can move all of these into the map function in the other file and define them off of the data of each item

Reply:
i wil look into this shortly